packages/commons/src/router/zodiosValidationErrorHandler.ts

```diff
@@ -0,0 +1,32 @@
+import { constants } from "http2";
+import { Request, Response, NextFunction } from "express";
+import { badRequestError, makeApiProblemBuilder } from "pagopa-interop-models";
+import { z } from "zod";
+import { fromZodIssue } from "zod-validation-error";
+import { logger } from "../logging/index.js";
+
+const makeApiProblem = makeApiProblemBuilder(logger, {});
+
+export function zodiosValidationErrorToApiProblem(
+  zodError: {
+    context: string;
+    error: z.ZodIssue[];
+  },
+  _req: Request,
+  res: Response,
+  _next: NextFunction
+): void {
+  const details = `${zodError.context} = ${zodError.error
```

Comment:
we could leverage on `errors` attribute of the Problem model, to obtain something like

```
{
    "type": "about:blank",
    "title": "Bad Request",
    "status": 400,
    "detail": `Incorrect value for ${zodError.context}`,
    "errors": [
        {
            "code": <theGenericErrorCode>,
            "detail": <Error description from zodError.error[0]>
        },
        {
            "code": <theGenericErrorCode>,
            "detail": <Error description from zodError.error[1]>
        },
        ...
    ]
}
```

Reply:
I can try, but this requires a refactor of how we build API problems around the app: at the moment, we always see only one `error`

Reply:
I propose to do it in a further PR so that this one is limited to fixing the issue using the error utilities we have available at the moment! What do you think?

Reply:
Makes sense, done! Now this is how the error looks like - I'm gonna update the screenshots in the PR description.

<img width="649" alt="image" src="https://github.com/pagopa/interop-be-monorepo/assets/6418684/8faa3698-e2ca-44e6-9e1a-f3e37ff00355">

Reply:
great, thank you!